### PR TITLE
[miio] allow for device remarks in the readme from json db

### PR DESCRIPTION
**Pull request by marcelrv**
Signed-off-by: Marcel Verpaalen <marcel@verpaalen.com>

### CODE REVIEW

File: bundles/org.openhab.binding.miio/src/test/java/org/openhab/binding/miio/internal/ReadmeHelper.java [hunk 1]
```diff
@@ -99,7 +111,7 @@ private StringWriter deviceList() {
                 sw.write(minLengthString(model, 22));
                 sw.write(" | ");
                 sw.write(isSupported ? "No       " : "Yes      ");
-                sw.write(" |            |\r\n");
+                sw.write(" | " + minLengthString(remark, 10) + " |\r\n");
```
Comment (cpmeister) [inline bundles/org.openhab.binding.miio/src/test/java/org/openhab/binding/miio/internal/ReadmeHelper.java:114]:
```suggestion
                sw.write(" | ");
                sw.write(minLengthString(remark, 10));
                sw.write(" |\r\n");
```

**Review by cpmeister — CHANGES_REQUESTED** [review]

**Review by cpmeister — APPROVED** [review]
LGTM

**Comment by Hilbrand** [comment]
Please don't create branches in the repository!!!! And if you do clean up after it's merged.

**Comment by marcelrv** [comment]
@Hilbrand I normally don't... it was wrong setting in my newly created OH3 eclipse instance.

**Comment by kaikreuzer** [comment]
Is there anything we need to fix in the IDE setup?

**Comment by marcelrv** [comment]
@kaikreuzer I'll do some checking this weekend, maybe setup a new one.
I had quite some difficulties getting my oh3 eclipse going, so maybe in my attempts to fix it ,I setup git wrong.

**Pull request merged**